csharp/Profiler/ProfilerTracer.cs

```diff
@@ -67,6 +67,19 @@ public void Trace(IScriptExtent extent, ScriptBlock scriptBlock, int level, stri
                 : _tickDivider == 100
                     ? timestampRaw / 100
                     : (long)(timestampRaw / _tickDivider);
+
+            var workingSet = Environment.WorkingSet;
+            var heapSize = GC.GetTotalMemory(false);
+            var gc0 = GC.CollectionCount(0);
+            var gc1 = GC.CollectionCount(1);
+            var gc2 = GC.CollectionCount(2);
+
+#if NET5_0_OR_GREATER
+            var allocatedBytes = GC.GetTotalAllocatedBytes();
```

Comment:
This is why I upgraded to net6.0 libs. Will need to polyfill this for the older library.